drivers/spi/Kconfig.npcx

```diff
@@ -0,0 +1,20 @@
+# Nuvoton NPCX SPI Driver configuration options
+
+# Copyright (c) 2024 Nuvoton Technology Corporation.
+# SPDX-License-Identifier: Apache-2.0
+
+menuconfig SPI_NPCX_SPIP
+	bool "Nuvoton NPCX embedded controller (EC) SPI driver"
+	default y
+	depends on DT_HAS_NUVOTON_NPCX_SPIP_ENABLED
+	help
+	  Enable the SPI peripherals on NPCX MCU.
+
+if SPI_NPCX_SPIP
+
+config SPI_NPCX_SPIP_INTERRUPT
+	bool "NPCX SPIP Interrupt Support"
+	help
```

Comment:
use `depends on SPI_NPCX_SPIP` rather than an `if` block if it's the only option, unless you already know that you are going to add more